CodeLite/fileutils.cpp

```diff
@@ -62,6 +62,8 @@
 #include <memory>
 #include <wx/filename.h>
 
+static bool bRealPathModeResolveSymlinks = true;
```

Comment:
Would it make sense to promote the static class in non static one and have it as regular member?

Reply:
I'm ok with the current solution - but in the end it is up to Eranif is guess